subgraphs/playback/src/resolvers/helpers.ts

```diff
@@ -0,0 +1,42 @@
+import { OmitIndexSignature, ValueOf } from 'type-fest';
+import { GraphQLError } from 'graphql';
+// import { ContextValue } from '../types';
+import { ResolversParentTypes, Resolver, ResolverFn } from '../__generated__/resolvers-types';
+import { getFieldConfig, identify } from '../fieldConfigs';
+// import { wait } from '../utils/common';
+
+type ParentTypes = ValueOf<OmitIndexSignature<ResolversParentTypes>>;
+
+export function prop<TParent extends ParentTypes, TKey extends keyof TParent>(
+  key: TKey
+): Resolver<TParent[TKey], TParent> {
+  return (parent: TParent) => parent[key];
+}
+
+// exp0ort function itself<TParent extends ParentTypes>(): Resolver<
+//   TParent,
+//   TParent
+// > {
+//   return (parent: TParent) => parent;
+// }
+
+// export function wrapWithSynthetics<TResult, TParent = {}, TArgs = {}>(
```

Comment:
This one is super important as it gives us the ability to artificially slow down the query (also necessary for the talk Alessia and I are giving later this year!!). Could we make sure all the resolvers are wrapped with this?

Reply:
Added this back in, just had to make some small tweaks!